config

```diff
@@ -16,5 +16,22 @@ path /echo EchoHandler {}
 
 path /status StatusHandler {}
 
+path / ProxyHandler {
+	proxy_pass www.ucla.edu;
+}
+
+# This tests the proxy on another local server hosted at 8081 that you must start.
+path /test ProxyHandler {
+	proxy_pass localhost:8081/example;
+}
+
+path /example ProxyHandler {
+	proxy_pass www.example.com;
+}
+
+path /redirect ProxyHandler {
+	proxy_pass ucla.edu;
```

Comment:
It looks like right now you allow the port to be specified after a colon following the host name. It might make it simpler from a parsing perspective to just have a new line with the port. Up to you!